### PR TITLE
Feat/autorenew actions

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: client/app/billing/autoRenew/autorenew.html [hunk 1]
```diff
@@ -163,86 +186,122 @@
...
-                    data-on-click="$ctrl.cancelServiceResiliation($row)"
-                    data-text="{{:: 'autorenew_service_canceldelete_button' | translate }}">
+                    data-on-click="$ctrl.cancelServiceResiliation($row)">
+                    <span data-translate="billing_autorenew_service_canceldelete_button"></span>
                 </oui-action-menu-item>
                 <!-- /Service with termination asked by the customer -->
 
+                <oui-action-menu-item
+                    data-ng-if="$row.url"
+                    data-href="{{ $row.url }}">
```
Comment (JeremyDec) [inline client/app/billing/autoRenew/autorenew.html]: ?
```suggestion
                    data-href="{{:: $row.url }}">
```

File: client/app/billing/autoRenew/autorenew.service.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import {
-  AUTORENEW_EVENT, CONTRACTS_IDS, NIC_URL, SERVICE_EXPIRATION, SERVICE_STATUS,
+  AUTORENEW_EVENT, CONTRACTS_IDS, NIC_URL, SERVICE_EXPIRATION, SERVICE_STATUS, RENEW_URL,
```
Comment (JeremyDec) [inline client/app/billing/autoRenew/autorenew.service.js]:
```suggestion
  AUTORENEW_EVENT, CONTRACTS_IDS, NIC_URL, RENEW_URL, SERVICE_EXPIRATION, SERVICE_STATUS,
```

Comment (marie-j) [reply client/app/billing/autoRenew/autorenew.service.js]: Changes made in another PR :)

Comment (JeremyDec) [reply client/app/billing/autoRenew/autorenew.service.js]: Ok :)

**Pull request merged**